tests/test_generate_readme.py

```
@@ -0,0 +1,540 @@
+                "model": "CPH2747",
+                "history": [{"status": "current", "version": "16.0.3.501", "arb": 0, "major": 3, "minor": 0, "last_checked": "2026-01-01"}]
+            }
+        }
+
+        history_file = self.history_dir / "15_GLO.json"
+        with open(history_file, 'w') as f:
+            json.dump(history_data, f)
+
+        with patch('sys.argv', ['generate_readme.py', str(self.history_dir)]):
+            with patch('builtins.open', mock_open()) as mock_file:
+                with patch('builtins.print') as mock_print:
```

Ruff (0.14.14)

[error] Local variable `mock_print` is assigned to but never used

Remove assignment to unused variable `mock_print`

(F841)